vendor/vendor.json

```diff
@@ -11,8 +17,8 @@
 		{
 			"checksumSHA1": "+2yCNqbcf7VcavAptooQReTGiHY=",
 			"path": "github.com/apparentlymart/go-rundeck-api",
```

Comment:
why is Rundeck needed?

Reply:
`govendor add` strikes again :stuck_out_tongue:

I'm going to try this again against my fork using `govendor fetch`, and see if I get the same deps pulled in.